Meta/Lagom/CMakeLists.txt

```diff
@@ -63,7 +63,17 @@ endif()
 include(wasm_spec_tests)
 
 add_compile_options(-Wno-unknown-warning-option -Wno-literal-suffix -Wno-deprecated-copy)
-add_compile_options(-O2)
+if (OPTIMIZATION_FLAG STREQUAL "O0")
+    add_compile_options(-O0)
+elseif (OPTIMIZATION_FLAG STREQUAL "O1")
+    add_compile_options(-O1)
+elseif (OPTIMIZATION_FLAG STREQUAL "O2")
+    add_compile_options(-O2)
+elseif (OPTIMIZATION_FLAG STREQUAL "O3")
+    add_compile_options(-O3)
+else ()
+    add_compile_options(-O2)
```

Comment:
Isn't setting these flags what `CMAKE_BUILD_TYPE` does already? https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html

Reply:
https://github.com/SerenityOS/serenity/blob/master/CMakeLists.txt#L7

Reply:
anyway, there is a smarter way to do this,
you can just asign to the variable if it is not set or give it a standard value as an option
and then just add the string contained to the compile options (maybe even just the number), maybe with some error checking before hand, to ensure that it is a valid optimization level